### ISSUE TITLE
multiple speakers timer running in parallel

### ISSUE THREAD

**Issue by wh1pch81n** [wh1pch81n/ToastMasterTimer#62]
Cool bug.

prerequisitses
3 second delay is enabled

repro
1)Create a speaker.
2)tap it
3)press start
4) before the count down timer finishes, swipe to the right

actual
after a few moments the total time in the tableview row will start showing the time ticking away.  If you do this multiple times, you can have multiple timers running at once

expected
All timers should stop when the tableview appears.

Note.  this is kind of a minor bug.  Its actually kind of cool.  It means I can do multiple timers at once.  However, this could be potentially bad when you start getting into speech recordings.  Only one speaker will get to use the speaker. another issue is... how do i stop the timer?

yes, this state prevents the timer from stopping.
